scss/components/_reboot.scss

```diff
@@ -13,3 +13,8 @@ html {
 body {
   min-width: $layout-min-width;
 }
+
+dt {
```

Comment:
Please don't do this. A **description list** in bootstrap is a responsive component and thus has two states. Forcing the **definition term** to always be right aligned will create issues.
In this case, to achieve the desired aspect, one should check the **responsive breakpoints** for the **description list** component and include the required **utility class for text alignment** already defined [here](https://getbootstrap.com/docs/4.1/utilities/text/#text-alignment) only when the design permits it (if the **definition term** and **definition description** are displayed inline).

Reply:
You're right, this is a bad idea. It makes more sense to just use utility classes. I'll remove it.